tests/test_utils_python.py

```diff
@@ -239,8 +239,11 @@ def __call__(self, a, b, c):
         self.assertEqual(get_func_args(" ".join, stripself=True), ["iterable"])
 
         if platform.python_implementation() == "CPython":
-            # doesn't work on CPython: https://bugs.python.org/issue42785
-            self.assertEqual(get_func_args(operator.itemgetter(2)), [])
+            # This never worked on CPython: https://bugs.python.org/issue42785
```

Comment:
Well it didn't work but now works :) How about "This didn't work on older versions of CPython"?

Also I would update the issue URL to https://github.com/python/cpython/issues/86951